src/main/java/com/flightstats/hub/dao/aws/AwsConnectorFactory.java

```diff
@@ -93,15 +93,16 @@ public AmazonDynamoDB getDynamoClient() {
     private AWSCredentialsProviderChain getAwsCredentials() {
         return new AWSCredentialsProviderChain(
                 new DefaultAWSCredentialsProviderChain(),
-                new AWSStaticCredentialsProvider(loadTestCredentials(awsProperties.getCredentialsFile())));
+                new AWSStaticCredentialsProvider(loadTestCredentials(awsProperties.getCredentialsFile()))
+        );
```

Comment:
what is the difference here

Reply:
I just re-aligned with the brackets. You can see I just moved 1 closing bracket down and kept that colon beside it.

Original:

new AWSStaticCredentialsProvider(loadTestCredentials(awsProperties.getCredentialsFile())));

Modified:

new AWSStaticCredentialsProvider(loadTestCredentials(awsProperties.getCredentialsFile()))
        );
        
However it'not impacting anything, just aligned.